Contest_API.md

```diff
@@ -143,24 +143,24 @@ their expected behavior, if implemented.
 Standard [HTTP status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) are
 returned to indicate success or failure. If successful DELETE will have no response body,
 GET on a collection will return the collection, and every other method will contain the
-current (updated) state of the element.
+current (updated) state of the object.
 
-If a POST, PUT, PATCH would cause any of the following issues it must fail, in addition to any endpoint or element-specific requirements:
+If a POST, PUT, PATCH would cause any of the following issues it must fail, in addition to any endpoint or object-specific requirements:
```

Comment:
Would "endpoint-specific" be better?

Reply:
It already says that: "endpoint or object-specific requirements"

Reply:
Indeed.